src/rpc/RPCHelpers.cpp

```diff
@@ -189,13 +189,209 @@ toJson(ripple::STBase const& obj)
     return value.as_object();
 }
 
+std::optional<ripple::uint256>
```

Comment:
The helper functions are mainly from https://github.com/XRPLF/rippled/blob/develop/src/ripple/rpc/NFTokenID.h which is not in libxrpl's scope.